integration-tests/src/tests/features/adversarial_behaviors.rs

```diff
@@ -235,7 +231,7 @@ fn test_banning_chunk_producer_when_seeing_invalid_chunk_base(
         assert_eq!(block.header().height(), height);
 
         let mut invalid_chunks_in_this_block: HashSet<ShardId> = HashSet::new();
-        let mut this_block_should_be_skipped = false;
+        let this_block_should_be_skipped = false;
```

Comment:
Can we remove this variable altogether and substitute occurrences with `false`?

Reply:
done.